cfg/builder/builder_walk.cc

```diff
@@ -344,11 +344,20 @@ BasicBlock *CFGBuilder::walk(CFGContext cctx, ast::Expression *what, BasicBlock
                                 return;
                             }
 
-                            if (ast::isa_tree<ast::Send>(s->args[0])) {
-                                // Providing a send is the most common way T.absurd is misused
+                            if (!ast::isa_tree<ast::Local>(s->args[0]) &&
+                                !ast::isa_tree<ast::UnresolvedIdent>(s->args[0])) {
```

Comment:
Wasn't sure from #3462 if we just want to allow just local variables, or if we also want to allow global/class/instance vars. Conceptually it seems reasonable to me to allow all four.